src/lib.rs

```diff
@@ -326,14 +308,6 @@ mod tests {
         FromHex::from_hex(s).expect("static str should contain valid hex bytes")
     }
 
-    #[test]
-    fn bn128_pairing_empty() {
-        // should not fail, because empty input is a valid input of 0 elements
```

Comment:
Cannot just remove this test. Need to add back a test then which provides empty input and expects `0x0000000000000000000000000000000000000000000000000000000000000001` as the output.

Reply:
ups.. yeah.. sorry - I really don't know why I thought I can just remove it 😄 so its in again 😄